packages/core/bootstrap/src/index.ts

```diff
@@ -79,13 +80,39 @@ const withStatusCode: Middleware = async (execute, context) => async (input) =>
 
 // Log adapter input & output data
 const withLogger: Middleware = async (execute, context) => async (input: AdapterRequest) => {
+  interface IError {
+    jobRunID: string
+    params: AdapterRequestData
+    message: string
+    feedID: string
+    endpoit?: string
+    errorResponse?: any
+    rawError?: any
+  }
   Logger.debug('Input: ', { input })
   try {
     const result = await execute(input, context)
     Logger.debug(`Output: [${result.statusCode}]: `, { output: result.data })
     return result
   } catch (error) {
-    Logger.error(error.toString(), { stack: error.stack })
+    const feedID = metrics.util.getFeedId(input)
+    const errorLog: IError = {
+      message: error.toString(),
+      jobRunID: input.id,
+      params: input.data,
+      feedID,
+    }
+    if (error.endpoint) {
+      errorLog.endpoit = error.endpoint
+    }
+    if (error.errorResponse) {
+      errorLog.errorResponse = error.errorResponse
+    }
```

Comment:
I think we should just log whatever the value is, even if it's falsy. Conditionally logging certain values provides a less complete picture when trying to debug with the logs.

Reply:
In many cases either endpoint or errorResponse will be undefined. Not sure if we intentionally should include this in logs